### PR TITLE
Revamp Blaze banner

### PR DESCRIPTION
**Pull request by timur987**
Related to https://jira.tumblr.net/browse/SELFSERVE-411

## Description
We want to revamp the banner according to a [Figma](https://www.figma.com/file/uvh2KtY65b7hGBHI9exbJ1/BlazePress?node-id=5448-44706&t=VXEiEfmXBAA0C9O1-0) design.

## Proposed Changes
* Change layout for promote posts / campaigns only
* Revamp a banner

## Testing Instructions

* Checkout this branch in you local environment
* Run Calypso (`yarn && yarn start`)
* Make sure you have less than 3 campaigns, otherwise you need to hack this [condition](https://github.com/Automattic/wp-calypso/pull/76094/files#diff-af45fef67dabb389ee3b79b06fd228c64fa0c2e369993c26b3e105c2b1e2cc04R277)
* Ensure you see the update banner
* Try out both posts and campaigns pages and make sure they look fine (I didn't make it responsive in a decent way, so it will be done down the road)

### Banner is revamped
blaze-press-new-banner

### Updated Layout
blaze-press-new-layout

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/promote-post-i2/main.tsx [hunk 1]
```diff
@@ -266,15 +258,25 @@ export default function PromotedPosts( { tab }: Props ) {
...
 			<DocumentHead title={ translate( 'Advertising - Redesign page!' ) } />
 
-			<FormattedHeader
-				brandFont
-				className="advertising__page-header"
-				headerText={ `${ translate( 'Advertising' ) } - Redesign page` }
-				subHeaderText={ campaignsData?.length ? subtitle : '' }
-				align="left"
-			/>
+			<div className="promote-post__top-bar">
```
Comment (sbarbosa) [inline client/my-sites/promote-post-i2/main.tsx]: We need to take into account that the style of the header differs if we are showing the banner or not.
From what I can see in the designs, the font size and styling are a little different if the banner is hidden, and a description appears below saying:  `Use Blaze to grow your audience by promoting your content across Tumblr and WordPress.com.`

**Review by sbarbosa — COMMENTED** [review]
It's looking great! 

I left a comment regarding the header. If we are hiding the banner, the header changes a little by increasing the font size and adding a description below.

File: client/my-sites/promote-post-i2/main.tsx (hunk 1 above)
Comment (timur987) [reply client/my-sites/promote-post-i2/main.tsx]: Great catch, @sbarbosa! 🏆 

Fixed in https://github.com/Automattic/wp-calypso/pull/76094/commits/f72e4d87a3838bbf594d1945701a3aa00237f51a. Here's how it look like:

### With a banner
Page title (banner exists)

### Without a banner
Page title (banner is hidden)

**Comment by sbarbosa** [comment]
Hi @timur987, the last changes are looking great! 
I found two small details regarding the layout for the table loading and the campaigns list page.

Screenshot 2023-05-02 at 12 33 47
Screenshot 2023-05-02 at 12 34 01

I did a little testing locally, and the fix appears to be easy; I will leave a patch in case you want to use it: [Banner.patch](https://github.com/Automattic/wp-calypso/files/11375116/Banner.patch).

**Comment by timur987** [comment]
Thanks for the patch, @sbarbosa! It works like a charm 👍

**Review by sbarbosa — APPROVED** [review]
Nice, LGTM 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7967341

Thank you @timur987 for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7967341).

**Pull request merged**